File: 9лаб_16варКЗП/лаб9/Speaker.py
# Клас Speaker представляє динамік із можливістю керування гучністю.
class Speaker:
    def __init__(self, volume):
        # Ініціалізує динамік із початковою гучністю.
        self.volume = volume

    # Метод для отримання поточного рівня гучності.
    def get_volume(self):
        return self.volume

    # Метод для встановлення нового рівня гучності.
    def set_volume(self, volume):
        self.volume = volume

    # Метод для формування рядкового представлення об'єкта Speaker.
    def __str__(self):
        # Повертає інформацію про поточний рівень гучності динаміка.
        return f"Speaker{{volume={self.volume}}}"

File: 3лаб_16варКЗП/лаб3/Lab3/src/KI/Milian/Lab3/Speaker.java
package KI.Milian.Lab3;

/**
 * Клас, що описує динамік аудіоплеєра.
 */
public class Speaker {
    private int volume; // Гучність динаміка

    public Speaker(int volume) {
        this.volume = volume;
    }

    public int getVolume() {
        return volume;
    }

    public void setVolume(int volume) {
        this.volume = volume;
    }

    @Override
    public String toString() {
        return "Speaker{volume=" + volume + "}";
    }
}